src/app.ts

```diff
@@ -55,8 +56,10 @@ export const setupApp = async ({ mongoClient }: AppSettings) => {
   app.use(reqHandler);
   app.use('/builds', buildsRouter);
   app.use('/projects', projectsRouter);
+  app.use('/user', userRouter);
   app.use('/prod/builds', buildsRouter);
   app.use('/prod/projects', projectsRouter);
+  app.use('/prod/user', userRouter);
```

Comment:
I'm unsure if we would ever need to support a `/prod` route for this since the Autobuilder only ever adds "docs-builder-bot" as the user for production deployments. Is it worth adding?

Reply:
Ah, I guess not! I was unsure of exactly what was using this, but it's just the chatbot and/or any future public needed API?

Reply:
The `/prod` endpoints are typically expected to be used by the chatbot for production deployment data. I'd be in favor of omitting this for now to avoid needing to keep track of an extra endpoint, and adding it back in the future if we ever have a concept of more users (or more endpoints for users?)